wordpress-mobile/WordPress-iOS
# Improves BasePageListCell.

Fixes #5249 and part of #5250 

To test:
1. Launch the app.
2. Open a blog with pages.
3. Open the list of pages for that blog.
4. Make sure the page cell looks right, interact with it.
5. Trash the post, make sure the "undo" cell looks right, interact with it.

Needs review: @frosty
